helm-charts/scripts/install-toolchain.sh

```diff
@@ -26,9 +26,10 @@ mkdir -p "${TOOLS_DIR}"
 HELM_VERSION="v3.4.1"
 KUBECTL_VERSION=v1.18.9
 KIND_VERSION=v0.5.1
+EKSD_VERSION=kubernetes-1-18
```

Comment:
I think the standard from the eks-d world would be to use RELEASE_BRANCH=1-18

Reply:
I was gonna say the same

Reply:
will change it 👍 
should we also switch to 1-19?

Reply:
Yes I think we should be fore-runners in adopting our 1-19 release artifacts :wink:

Reply:
changed it and also added one for RELEASE